src/FSharpPlus/Validations.fs

```diff
@@ -0,0 +1,137 @@
+module FSharpPlus.Validations
```

Comment:
I think we should discuss the names.
My impression is that this should be in the ``.Data`` namespace.
I'm not sure if we should call it ``AccValidation`` or just ``Validation``.

Reply:
Let's call it `Validation`